modules/api/src/main/java/net/countercraft/movecraft/MovecraftRepair.java

```diff
@@ -477,4 +486,68 @@ public static void initialize(Plugin plugin){
     public static MovecraftRepair getInstance() {
         return instance;
     }
+
+    public int convertOldCraftRepairStates(){
+        //Check in the old RepairStates folder
+        File repairStateDir = new File(plugin.getDataFolder(),"RepairStates");
+        if (!repairStateDir.exists()){
+            return 0;
+        }
+        int convertedRepairStates = 0;
+        File[] repairStates = repairStateDir.listFiles();
+        if (repairStateDir.exists() && (repairStates != null || repairStates.length > 0)){
+            for (File rs : repairStates){
+                if (!rs.getName().contains(".schematic")){
+                    continue;
+                }
+                String fileName = rs.getName();
+                fileName = fileName.replace(".schematic", "");
+                OfflinePlayer owner = null;
+                if (fileName.contains("_")) {
+                    String[] parts = fileName.split("_");
+                    owner = Bukkit.getPlayer(parts[0]) == null ? Bukkit.getOfflinePlayer(parts[0]) : Bukkit.getPlayer(parts[0]);
+                    fileName = "";
+                    if (parts.length > 2){
+                        for (int i = 1 ; i <= parts.length - 1 ; i++){
+                            fileName += parts[i];
+                            if (i < parts.length - 1){
+                                fileName += "_";
+                            }
+                        }
+                    } else {
+                        fileName = parts[1];
+                    }
+                }
+
+                for (Player p : Bukkit.getOnlinePlayers()){
```

Comment:
there will never be online players during startup, which is the only time this loop is run